states/huntsman/calibrating.py

```diff
@@ -106,7 +106,7 @@ def on_enter(event_data):
         pocs.say("Coarse focusing all cameras before starting observing for the night")
         autofocus_events = pocs.observatory.autofocus_cameras(coarse=True)
         pocs.logger.debug("Started focus, going to wait")
-        pocs.wait_for_events(autofocus_events.values(), 600)  # Longer timeout?
+        pocs.wait_for_events(list(autofocus_events.values()), 600)  # Longer timeout?
```

Comment:
So we shouldn't need the explicit `list` any more, is it not working if you leave it out?

Reply:
We still need it for now. You fixed this in the `listify` in `panoptes-utils` but `pocs.core.POCS.wait_for_events()` still uses the version from `pocs.utils`, which can't handle `dict_values` objects. Once POCS switches to `panoptes-utils` we can just pass the `dict` here.

Reply:
Oh, right. Maybe the fourth time you mention that I'll actually remember. :laughing: